app/about/page.tsx

```diff
@@ -1,5 +1,18 @@
 import React from "react"
 
+const communityDescription = `
+	We're a community of developers of all skill levels, dedicated to fostering a fun and
+	educational environment. Hosted by Sam Holmes and a team of passionate organizers, our
+	monthly meetups offer an opportunity to network, learn, and showcase your projects. At
+	each event, you'll enjoy complimentary food and drinks during our networking lunch,
+	followed by a series of engaging presentations on various developer and engineering
+	topics. After the talks, we break into groups for casual networking, project showcases,
+	and coding help. Whether you're a seasoned developer or just starting out, there's
+	something for everyone. Be sure to bring your laptop if you'd like to share your latest
+	project or give a presentation. We look forward to meeting you and seeing what you're
+	excited about!
+`
```

Comment:
I think I resolved this same issue in my PR, but instead of factoring out the string, I kept it inline. That'd be preferable since this consts is single-use.